sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -10595,22 +10594,22 @@
               "input": true,
               "tableView": true,
               "inputType": "radio",
-              "label": "<strong>Are you a student with disabilities and wish to be assessed for disability funding?</strong>",
+              "label": "Are you a student who has verified your Disability Status with StudentAid BC as a Permanent Disability (PD) or a Persistent or Prolonged Disability (PPD)?<br><span style=\"font-weight: normal\"> If so, choose option 1.<br> If you have not verified your Disability Status as PD or PPD before, and wish to receive other funding types, choose option 2, complete and submit an Appendix 8 form to StudentAid BC.This may allow you to receive some funding while StudentAid BC verifies your Disability Status. <br> You can find PD and PPD definitions, available disability type funding, eligibility requirements and how to submit your form on <a href=\"https://studentaidbc.ca/sites/all/files/form-library/appendix_8.pdf\" target=\"_blank\" class=\"formio-href\" rel=\"noopener noreferrer\">Appendix 8.</a></span>",
```

Comment:
should this be a part of this qn,
![image](https://github.com/bcgov/SIMS/assets/77353155/b1286c02-c61f-4584-bfe7-126bd7f312a6)
 I believe it can be added as a separate HTML component, as we do for other fields, maybe it will come under the options and we can avoid this lengthy qn in the validation,
![image](https://github.com/bcgov/SIMS/assets/77353155/e923c573-8d04-4abf-ba2e-c13ef543e12c)
![image](https://github.com/bcgov/SIMS/assets/77353155/ee1e3f18-e0b8-4905-98d2-410f8c23ba5c)

Reply:
should this be a part of this qn  - Yes according to the story @ann-aot, but I see your point that this is more a helper text for the question and not the question itself. 

I tried adding that in a separate HTML content and it looked like this(the mandatory asterisks was on all options)

![image](https://github.com/bcgov/SIMS/assets/54600590/118fd59b-8e98-4ad9-aadf-d058c6df0d4c)

And then I went back to the approach what I followed now. 

I will check with @HRAGANBC quickly explaining the impact of having lengthy question vs the HTML approach and see what can be done.

Reply:
@dheepak-aot, I was thinking about something like this, 
![image](https://github.com/bcgov/SIMS/assets/77353155/15e45032-7c82-42a5-acec-ee07f5c6fd81)